NitroxServer/GameLogic/PlayerManager.cs

```diff
@@ -123,6 +123,30 @@ public void PlayerDisconnected(Connection connection)
             }
         }
 
+
+        /// <summary>
+        /// Trys to return a player by name
+        /// </summary>
+        /// <param name="playerName">The player to find</param>
+        /// <param name="foundPlayer">Will be null if return value is false</param>
+        /// <returns>true if the player was found false otherwise</returns>
```

Comment:
minor: commenting doesn't add value.  Summary is the name of the method, parameters are self explainable, and the usage aligns with c#'s Try* that is used everywhere in the code.